Resources/Prototypes/Actions/types.yml

```diff
@@ -161,6 +168,7 @@
     event: !type:UseDnaScramblerImplantEvent
 
 - type: entity
+  parent: BaseGeraAction
```

Comment:
theres no other actions using it so why have a parent like this

Reply:
No reason it seems